packages/account/src/Components/personal-details/__tests__/personal-details.spec.js

```diff
@@ -98,7 +94,12 @@ const runCommonFormfieldsTests = () => {
         'https://www.oecd.org/tax/automatic-exchange/crs-implementation-and-assistance/tax-identification-numbers/'
     );
 
-    expect(screen.getByRole('heading', { name: /account opening reason/i })).toBeInTheDocument();
+    if (is_svg)
```

Comment:
nit: May be we can write a sepeaate testcase next time rather than adding a expect inside if condition

Reply:
Sure thing. Thanks @likhith-deriv

Reply:
But I had to add this condition as now we've this primary condition in our component so some of the test cases had to be handled